Lmod/branches/7.0/z01-default_modules.sh

```diff
@@ -0,0 +1,11 @@
+if [ -z "$__Init_Default_Modules" -o -z "$LD_LIBRARY_PATH" ]; then
+   export __Init_Default_Modules=1;
+   if [ -f "/usr/share/modulefiles/DefaultModules.lua" ];then
+     if [ -z "$LMOD_SYSTEM_DEFAULT_MODULES" ];then
+       export LMOD_SYSTEM_DEFAULT_MODULES="DefaultModules"
```

Comment:
fyi. discussed: reverse L3/L4 conditionals, and an else clause specifying `StdEnv` if all other conditions fail!